public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/helpers.tsx

```diff
@@ -44,18 +47,40 @@ const getFeatureNameFromVisParams = (id, seriesParams) => {
 };
 
 function visAggregationToFormik(value) {
-  return [
-    {
-      label: value.params.field.name,
-      type: 'number',
-    },
-  ];
+  if (Object.values(value.params).length !== 0) {
```

Comment:
can we add a regression test for this?

Reply:
sure, will add a test case for this later. Looked up react best practice guidance. It seems like this is the best practice on how to check if an array is empty as react doesn't have built-in Array.isEmpty() method

Reply:
I just meant adding a test where this conditional is false. Also, you may use lodash `isEmpty()` fn that we use other places in the plugin